### ISSUE TITLE
TestFlight v10.306.2: Unicode characters in paste result in mojibake

### ISSUE THREAD

**Issue by treyharris** [blinksh/blink#507]
This problem is newly-arisen in the TestFlight version of Blink; the App Store version does not exhibit this behavior. It is possibly a regression but I have not attempted this in other versions to find out.

### Steps to reproduce:

1. Select and copy the following text block from an iOS browser:
```
Certain text, such as letters with diacritics(åéîøüùñ), elipses (…), dashes (—, –) and other non-ASCII symbols (e.g., × † ÷) get pasted incorrectly.¹
—
¹ Unlike in prior versions of Blink.
```
2. Open a Blink ssh or mosh session to a host capable of handling UTF-8 input and output streams.
3. Execute a program able to accept free input (such as an editor or `cat > /dev/null`).
4. Paste (with ⌘ V or the long-press menu item).

### Expected results:
 The following text pasted:
```
Certain text, such as letters with diacritics(åéîøüùñ), elipses (…), dashes (—, –) and other non-ASCII symbols (e.g., × † ÷) get pasted incorrectly.¹
—
¹ Unlike in prior versions of Blink.
```

### Actual results:

Mojibake in the pasted results, like:
```
Certain text, such as letters with diacritics(Ã¥Ã©Ã®Ã¸Ã¼Ã¹Ã±), elipses (â¦), dashes (â, â) and other non-ASCII symbols (e.g., Ã â  Ã·) get pasted incorrectly.Â¹                                    
â                                                                                                                                                                                        

Â¹ Unlike in prior versions of Blink.
```

Note that _copying_ Unicode characters _from_ Blink and pasting into another iOS app seems to work correctly.

But even copying these characters off the Blink terminal (if you can produce them another way) and immediately pasting them back results in the mojibake. This seems to indicate an NFC/NFD normalization mismatch.

**Comment by yury** [comment]
Hi @treyharris,

Thank you for your feedback. Fix will be on next release.

**Comment by treyharris** [comment]
Can confirm fix works in today's TestFlight 10.329 (1).

**Comment by yury** [comment]
Thank you. Closing.
